modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
```

Comment:
If you update the `id` and `=` title below, the `:description:` (line 2) and `:navtitle:` (line 4) values have to be updated too to be the same as the `=` title.